homework.md

```diff
@@ -98,16 +101,16 @@ Send you code to Github
 
 What program do we use to write code at CodeYourFuture?
 
-Microsoft Word
+visual studio
```

Comment:
Slight difference; we use Visual Studio _Code_ (VSCode), which is different to Visual Studio (another program you can use to write code, mainly C#)